api/envoy/service/status/v2/csds.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+package envoy.service.status.v2;
+
+import "envoy/admin/v2alpha/config_dump.proto";
+import "envoy/api/v2/core/base.proto";
+import "envoy/type/matcher/metadata.proto";
+import "envoy/type/matcher/string.proto";
+
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+
+import "envoy/annotations/resource.proto";
+
+option java_package = "io.envoyproxy.envoy.service.status.v2";
+option java_outer_classname = "CsdsProto";
+option java_multiple_files = true;
+option java_generic_services = true;
+
+// CSDS is Client Status Discovery Service. It can be used to get the status of
+// xDS-compliant client from the management server's point of view. In the
+// future, it can also potentially be used as an interface to get the current
+// state directly from the client.
+service ClientStatusService {
+  option (envoy.annotations.resource).type = "envoy.service.status.v2.ClientStatus";
```

Comment:
This isn't needed if you're not using `Discovery{Request,Response}`.

Reply:
Done.